From 1e19f6017da9c33569d91cbb54e231fe80785ba1 Mon Sep 17 00:00:00 2001
From: mldn <mldnqa@163.com>
Date: Sat, 23 May 2015 23:40:13 +0800
Subject: [PATCH 1/2] =?UTF-8?q?=E5=9C=A8House.java=E9=87=8C=E5=A2=9E?=
 =?UTF-8?q?=E5=8A=A0=E4=BA=86toString()?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 House.java | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/House.java b/House.java
index 05d6275..dbeaba7 100644
--- a/House.java
+++ b/House.java
@@ -6,5 +6,9 @@ public class House{
 		System.out.println("Hello") ;
 		System.out.println("Hello") ;
 		System.out.println("Hello") ;
+
+		public String toString(){
+			
+		}
 	}
 }
\ No newline at end of file
-- 
1.9.5.msysgit.0

